mergeobjects.py

```diff
@@ -0,0 +1,126 @@
+# coding=utf-8
+
+"""
+MergeObjects
+===========
+
+**MergeObjects** merges objects below a certain threshold into its most prevalent, adjacent neighbor.
+
+The output of this module is a object image of the same data type as the input.
+**MergeObjects** can be run *after* any labeling or segmentation module (e.g.,
+**ConvertImageToObjects** or **Watershed**). Labels are preserved and, where possible, small
+objects are merged into neighboring objects that constitute a majority of the small object's
+border. This is useful for reversing over-segmentation and artifacts that result from seeding
```

Comment:
Would you be interested in contributing a seeding module here? /cc @karhohs

Reply:
Oops I didn't realize this referenced modules that didn't exist in either repo. @karhohs wrote the seeding modules we use I believe, but I can add them to the plugins if he doesn't mind. We've also converted them to `ObjectProcessing` modules from `ImageProcessing` ones.